osu.Game.Tests/Visual/UserInterface/TestSceneNowPlayingOverlay.cs

```diff
@@ -44,21 +44,42 @@ public void TestShowHideDisable()
             AddStep(@"hide", () => nowPlayingOverlay.Hide());
         }
 
+        [Resolved]
+        private BeatmapManager manager { get; set; }
```

Comment:
This is super dodgy since it's importing maps into the global database. All other tests that import beatmaps use their own local database instead...

Also interestingly enough this shows a case that carousel isn't prepared for:
```
[runtime] 2020-04-28 03:31:46 [error]: An unhandled error has occurred.
[runtime] 2020-04-28 03:31:46 [error]: System.NullReferenceException: Object reference not set to an instance of an object.
[runtime] 2020-04-28 03:31:46 [error]: at osu.Game.Screens.Select.Carousel.CarouselBeatmap.<>c__DisplayClass3_0.<Filter>b__0(String term) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Select/Carousel/CarouselBeatmap.cs:line 64
[runtime] 2020-04-28 03:31:46 [error]: at System.Linq.Enumerable.Any[TSource](IEnumerable`1 source, Func`2 predicate)
[runtime] 2020-04-28 03:31:46 [error]: at osu.Game.Screens.Select.Carousel.CarouselBeatmap.Filter(FilterCriteria criteria) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Select/Carousel/CarouselBeatmap.cs:line 64
[runtime] 2020-04-28 03:31:46 [error]: at osu.Game.Screens.Select.Carousel.CarouselGroup.<>c__DisplayClass10_0.<Filter>b__0(CarouselItem c) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Select/Carousel/CarouselGroup.cs:line 85
[runtime] 2020-04-28 03:31:46 [error]: at System.Collections.Generic.List`1.ForEach(Action`1 action)
[runtime] 2020-04-28 03:31:46 [error]: at osu.Game.Screens.Select.Carousel.CarouselGroup.Filter(FilterCriteria criteria) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Select/Carousel/CarouselGroup.cs:line 85
[runtime] 2020-04-28 03:31:46 [error]: at osu.Game.Screens.Select.Carousel.CarouselGroupEagerSelect.Filter(FilterCriteria criteria) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Select/Carousel/CarouselGroupEagerSelect.cs:line 43
[runtime] 2020-04-28 03:31:46 [error]: at osu.Game.Screens.Select.Carousel.CarouselBeatmapSet.Filter(FilterCriteria criteria) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Select/Carousel/CarouselBeatmapSet.cs:line 94
[runtime] 2020-04-28 03:31:46 [error]: at osu.Game.Screens.Select.Carousel.CarouselGroup.<>c__DisplayClass10_0.<Filter>b__0(CarouselItem c) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Select/Carousel/CarouselGroup.cs:line 85
[runtime] 2020-04-28 03:31:46 [error]: at System.Collections.Generic.List`1.ForEach(Action`1 action)
[runtime] 2020-04-28 03:31:46 [error]: at osu.Game.Screens.Select.Carousel.CarouselGroup.Filter(FilterCriteria criteria) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Select/Carousel/CarouselGroup.cs:line 85
[runtime] 2020-04-28 03:31:46 [error]: at osu.Game.Screens.Select.Carousel.CarouselGroupEagerSelect.Filter(FilterCriteria criteria) in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Select/Carousel/CarouselGroupEagerSelect.cs:line 43
[runtime] 2020-04-28 03:31:46 [error]: at osu.Game.Screens.Select.BeatmapCarousel.<>c__DisplayClass74_0.<applyActiveCriteria>g__perform|0() in /Users/smoogipoo/Repos/osu/osu.Game/Screens/Select/BeatmapCarousel.cs:line 423
[runtime] 2020-04-28 03:31:46 [error]: at osu.Framework.Threading.ScheduledDelegate.RunTaskInternal()
```

Reply:
Interesting, which test are you seeing that on?

Reply:
Running this test, then starting up osu.Desktop and searching "title".